docs/mediapipe.md

```diff
@@ -262,3 +265,18 @@ in the conditions:default section of the deps property:
 - Updates in subconfig files and mediapipe graph files do not trigger model server config reloads. The reload of the full config, including subconfig and graphs, can be initiated by an updated in the main config json file or using the REST API `config/reload` endpoint. 
 
 
+## Known issues <a name="known-issues"></a>
+- MediaPipe `SyncSetInputStreamHandler` options are not validated during graph validation, but graph creation:
+```
+input_stream_handler {
+    input_stream_handler: "SyncSetInputStreamHandler",
+    options {
+        [mediapipe.SyncSetInputStreamHandlerOptions.ext] {
+            sync_set {
+                tag_index: "LOOPBACK:1"  # wrong index example
+            }
+        }
+    }
+}
+```
+MediaPipe closes entire application when the index does not match. Therefore test the deployment for correct setting before deploying on production.
```

Comment:
```suggestion
MediaPipe closes entire application because there's no input stream with tag `LOOPBACK` and index `1` (correct index is `0`). Therefore, test the deployment for correct setting before deploying to production.
```

Reply:
done